### PR TITLE
additional include to override vars

### PR DESCRIPTION
**Pull request by stephenpc**

### CODE REVIEW

**Review by ssdnex — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -12,14 +12,21 @@
...
+# Tag/Release/Branch of source repository for repo manifests
 REPO_BRANCH ?= main
-# this should point to a seed manifest
+# Path to seed manifest in repository
 REPO_MANIFEST ?= manifests/terraform_modules/seed/manifest.xml
 
-# Settings to pull in nexient version of (google) repo utility that supports environment substitution:
+# Settings to pull in Nexient version of (google) repo utility that supports environment substitution:
 REPO_URL ?= https://github.com/nexient-llc/git-repo.git
 REPO_REV ?= main
```
Comment (dresdencraft) [inline Makefile:31]: Add comment of the supported values

File: README.md [hunk 2]
```diff
@@ -73,6 +74,10 @@ TBD
 
 ### Repo Setup
 
+#### Overriding Make Behavior
+
+When run, `make` will look for a file called `.cafenv` in `$HOME` and the local directory. This file if present will be included, and it can be used to override variables without altering the [Makefile](Makefile). A few examples of variables that can be substituted are commented out in the file. If `.cafenv` exists in both `$HOME` and the local directory, values set in the file in the local directory will take precedence.
```
Comment (dresdencraft) [inline README.md:79]: update this to working dir

**Review by dresdencraft — CHANGES_REQUESTED** [review]
see in line comments

File: Makefile [hunk 3]
```diff
@@ -12,14 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# this will always point here
+# Include custom values from .cafenv. $HOME is searched first, followed by the
+# local directory. Values found in .cafenv if it exists in the local directory
+# take precedence.
+CAF_ENV_FILE = .cafenv
+-include $(HOME)/$(CAF_ENV_FILE)
```
[the remark this answers is not in the document]
Comment (ssdnex) [reply Makefile:19]: `repo` doesn't support tags at this point, only branches
https://source.android.com/docs/setup/create/repo#init

[the remark this answers is not in the document]
Comment (stephenpc) [reply Makefile:19]: @dresdencraft The line below (L20) assumes the root of the repository

[the remark this answers is not in the document]
Comment (stephenpc) [reply Makefile:19]: @dresdencraft Do you want to have ONLY the root of the repo be the location of this file?

**Comment by stephenpc** [comment]
@dresdencraft I've addressed the inline comments. Please re-review.

**Review by nexient-phil — APPROVED** [review]

**Pull request merged**